distribution/lib/Standard/Base/0.0.0-dev/src/Data/Read/Return_As.enso

```diff
@@ -32,16 +33,25 @@ type Return_As
     to_display_text self -> Text = self.underlying.to_display_text
 
     ## PRIVATE
-    make_return self (input : Many_Files_List) (objects : Vector Any) =
-        self.underlying.make_return input objects
+    make_return self (input : Many_Files_List) (objects : Vector Any) (on_problems : Problem_Behavior) =
+        self.underlying.make_return input objects on_problems
 
     ## PRIVATE
        Resolve an unresolved constructor to the actual type.
     private resolve value = case value of
         _ : Function ->
+            # Result when we fail to resolve an autoscoped constructor.
+            symbol_not_found =
+                # TODO ideally we'd like to use `Meta` but it currently has no support for UnresolvedConstructor so we use a 'hack'.
+                is_the_default = value.to_text == "..As_Merged_Table"
```

Comment:
Checking `to_text` feels like a bit of a hack, but currently `Meta` seems to lack support for `UnresolvedConstructor` - `Meta.meta` returns `Primitive` and does not allow any inspecting.

It feels like it may prove useful in the future, @jdunkerley do we want to create a ticket for the engine team to add support for `Unresolved_Constructor` in `Meta`?

Reply:
I would have done it via
```
type My_Private_Type
    As_Merged_Table
```
and trying to convert to that.

I agree we could probably do with something on Meta.
cc: @JaroslavTulach  for visibility.